### PR TITLE
Fix colour setting

### PR DESCRIPTION
**Pull request by akarol**
{{pytest: -v -k 'test_color_setting'}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit eb4fdee9f30d1b13ee823fb5e13f4e6293b3c2dd:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/configure/test_display_settings.py [hunk 1]
```diff
@@ -57,8 +57,7 @@ def test_timezone_setting(set_timezone):
     assert sel.is_displayed(locator), "Timezone settings Failed"
 
 
-@pytest.mark.ignore_stream("upstream")
-@pytest.mark.ignore_stream("5.4")
+@pytest.mark.ignore_stream("upstream", "5.4", "5.5")
```
Comment (psav) [inline cfme/tests/configure/test_display_settings.py]: Waiting on an uncollectif for versions > 5.4, then mergy mergy

**Comment by dajoRH** [comment]
Lint report for commit 933e5c233c73070a2fcd120404480f0b4b77f709:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit a10785cbe1a143089652accc37a9f46475872e58:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/configure/test_display_settings.py (hunk 1 above)
Comment (akarol) [inline cfme/tests/configure/test_display_settings.py]: @psav  Done!

**Pull request merged**